data/io.elementary.appcenter.policy.in

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1.0/policyconfig.dtd">
+<policyconfig>
+  <vendor>elementary</vendor>
+  <vendor_url>https://elementary.io/</vendor_url>
+
+  <action id="io.elementary.appcenter.update">
+    <description>Update software</description>
+    <message>Authentication is required to update software</message>
+    <icon_name>package-x-generic</icon_name>
```

Comment:
Where is this icon displayed? Should we use something like `software-update` here?

Reply:
It's not displayed anywhere in theory due to the fact that this is an automatically permitted action now. I just copied the text and icon lines from PackageKit so it's consistent.

It's possible that the dialog would still show in some circumstances like being on a vnc session instead of a local session. So feel free to change the icon to whatever you feel. I can't remember if the polkit dialog even uses it....